sources/packages/api/src-sql/Institution/Create-Institution.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS institutions(
+  id SERIAL PRIMARY KEY, 
+  legal_operating_name VARCHAR(64) NOT NULL, 
+  operating_name VARCHAR(64) NULL,  
+  primary_phone VARCHAR(64) NOT NULL,  
+  primary_email VARCHAR(64) NOT NULL,  
+  institution_type VARCHAR(64) NOT NULL,  
+  website VARCHAR(64) NOT NULL,  
+  regulating_body VARCHAR(64) NOT NULL,  
+  established_date  Date NOT NULL DEFAULT now(), 
+  primary_contact JSONB NOT NULL,
+  legal_authority_contact JSONB NOT NULL,  
+  institution_address JSONB NOT NULL,  
+  institution_mailing_address JSONB,
+  -- Reference Columns
+  user_id INT REFERENCES users(id) ON DELETE CASCADE,
```

Comment:
As per my understanding, the institution will not have a one-to-one relationship with the "users" table. Many BCeIDs will have access and "share the same institution record".
Maybe we need further discussions on this, but right now, one-to-one doesn't seem right to me.

Reply:
Yes we need discussion

Reply:
Correct "Many BCeIDs will have access and "share the same institution record". We need to meet to review further

Reply:
Done